src/Nethermind/Nethermind.Serialization.Json/EthereumJsonSerializer.cs

```diff
@@ -63,6 +65,7 @@ private static JsonSerializerOptions CreateOptions(bool indented, IEnumerable<Js
                 DictionaryKeyPolicy = JsonNamingPolicy.CamelCase,
                 PropertyNameCaseInsensitive = true,
                 MaxDepth = maxDepth,
+                Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping,
```

Comment:
Unfortunately any configurations that allow to only add a `+` sign are internal, so not accessible without reflection or copying framework code.

Potential risks are described [here](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/character-encoding#serialize-all-characters). Should be fine as client isn't intended to work with any HTML/JS and doesn't need XSS protection.

Reply:
Also we are the generators of it and we aren't adding XSS stuff